app/views/layouts/application.slim

```diff
@@ -1,7 +1,10 @@
 doctype html
-html
+html lang='en'
   head
     title = content_for?(:title) ? yield(:title) + " - #{site_title}" : site_title
+    // TODO: all pages should set a unique meta tag
+    - description = content_for?(:description) ? yield(:description) : "The worlds most advanced speedruning analytics and racing platform!"
```

Comment:
Did we want Frank to look at this description first before merging?

Reply:
Yeah sure

Reply:
@glacials were you able to get Frank's opinion on this this week?

Reply:
Yeah he said he was going to think on it, but that this is fine to go out as-is. I think `worlds` should be `world's` though.